privacyidea/lib/importotp.py

```diff
@@ -623,7 +626,10 @@ def export_pskc(tokenobj_list, psk=None):
             timedrift = 0
         otpkey = tokenobj.token.get_otpkey().getKey()
         try:
-            encrypted_otpkey = aes_encrypt_b64(psk, binascii.unhexlify(otpkey))
```

Comment:
Just that I understand correctly: Export of ``pw`` tokens previously caused an error in this line, becuase ``otpkey`` was not the hexlified password, but the actual password?

Reply:
yes. This resulted in no or very few PW tokens being exported.